com.ibm.streamsx.topology/opt/python/packages/streamsx/rest.py

```diff
@@ -41,26 +41,29 @@ class StreamsConnection:
     """
     def __init__(self, username=None, password=None, resource_url=None):
         # manually specify username, password, and resource_url
-        if username and password and resource_url:
+        if username and password:
+            # resource URL can be obtained via streamtool geturl or REST call
             pass
-        elif username and password and st._has_local_install:
-            resource_url = st.get_rest_api()
         elif st._has_local_install:
             # Assume quickstart
             username = 'streamsadmin'
             password = 'passw0rd'
-            resource_url = st.get_rest_api()
         else:
-            raise ValueError("Must supply either a BlueMix VCAP Services or a username, password, and resource url"
+            raise ValueError("Must supply either a BlueMix VCAP Services or a username, password"
```

Comment:
While you are in there ... it's "Bluemix", not "Blue**M**ix".

Also it's no longer a "StreamsContext" is it? (line 53)